ckan/lib/helpers.py

```diff
@@ -1094,7 +1095,7 @@ def build_extra_admin_nav():
 def _make_menu_item(menu_item, title, **kw):
     ''' build a navigation item used for example breadcrumbs
 
-    outputs <li><a href="..."></i> title</a></li>
+    outputs <li class="nav-item nav-link"><a href="..."></i> title</a></li>
```

Comment:
although a private function so not in the docs I believe, this output isn't shown as code. Could be updated while making this update.

outputs ``<li class="nav-item nav-link"><a href="..."></i> title</a></li>``